README.md

```diff
@@ -1535,6 +1538,96 @@ hermione_base_url=http://example.com hermione path/to/mytest.js
+- `--repl-before-test` - the same as `--repl` option except that REPL interface opens automatically before run test. Disabled by default;
+- `--repl-on-fail` - the same as `--repl` option except that REPL interface opens automatically on test fail. Disabled by default.
+
+#### switchToRepl
+
+Command that stops the test execution and opens REPL interface in order to communicate with browser. For example:
+
+```js
+it('foo', async ({browser}) => {
+    console.log('before open repl');
+
+    await browser.switchToRepl();
+
+    console.log('after open repl');
+});
+```
+
+And run it using the command:
+
+```bash
+npx hermione --repl --grep "foo" -b "chrome"
+```
+
+In this case, we are running only one test in one browser (or you can use `hermione.only.in('chrome')` before `it`).
+When executing the test, the text `before open repl` will be displayed in the console first, then test execution stops, REPL interface is opened and waits your commands. So we can write some command in the terminal:
+
+```bash
+await browser.getUrl();
+// about:blank
+```
+
+After the user closes the server, the test will continue to run (text `after open repl` will be displayed in the console and browser will close).
+
+Another command features:
+- all `const` and `let` declarations called in REPL mode are modified to `var` in runtime. This is done in order to be able to redefine created variables;
+- before switching to the REPL mode `process.cwd` is replaced with the path to the folder of the executed test. After exiting from the REPL mode `process.cwd` is restored. This feature allows you to import modules relative to the test correctly;
+- ability to pass the context to the REPL interface. For example:
+
+    ```js
+    it('foo', async ({browser}) => {
+        const foo = 1;
+
+        await browser.switchToRepl({foo});
+    });
+    ```
+
+    And now `foo` variable is available in REPL:
+
+    ```bash
+    console.log("foo:", foo);
+    // foo: 1
+    ```
+
+#### Test development in runtime
+
+For quick test development without restarting the test or the browser, you can to run the test in the terminal of IDE with enabled REPL mode:
+
+```bash
+npx hermione --repl-before-test --grep "foo" -b "chrome"
+```
+
+After that, you need to configure the hotkey in IDE to run the selected one or more lines of code in the terminal. As a result, each new written line can be sent to the terminal using a hotkey and due to this, you can write a test much faster.
+
+##### How to set up using VSCode
+
+1. Open `Code` -> `Settings...` -> `Keyboard Shortcuts` and print `run selected text` to search input. After that, you can specify the desired key combination
+2. Run hermione in repl mode (examples were above)
+3. Select one or mode lines of code and press created hotkey
+
+##### How to set up using Webstorm
+
+Ability to run selected text in terminal will be available after this [issue](https://youtrack.jetbrains.com/issue/WEB-49916/Debug-JS-file-selection).
```

Comment:
`after this issue` -> `after resolving this issue`,
`after this issue` -> `after this issue will be resolved`